src/test/java/org/example/repository/AccountRepositoryImplTest.java

```diff
@@ -0,0 +1,83 @@
+package org.example.repository;
+
+import org.example.connection.ConnectionSingletonTest;
+import org.example.enums.Card;
+import org.example.initDB.InitDBTest;
+import org.example.repository.dao.AccountRepository;
+import org.example.repository.dao.UserRepository;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static org.junit.Assert.*;
+
+public class AccountRepositoryImplTest {
+
+    private Connection connection;
+    private AccountRepository accountRepository;
+
+    @Before
+    public void init() {
+        connection = ConnectionSingletonTest.getConnection();
+        InitDBTest.runScript(connection);
+        accountRepository = new AccountRepositoryImpl(connection);
+    }
+
+    @After
+    public void end() throws SQLException {
+        connection.close();
+    }
+
+    @Test
+    public void create() {
+
+        String type = Card.VISA.getType();
+        double amount = 9999.0;
+        int accountId = accountRepository.create(type, amount);
+
+        Assert.assertTrue(accountId != -1);
+        Assert.assertEquals(amount, accountRepository.get(accountId), 0.001);
+    }
+
+    @Test
+    public void get() {
+
+        int expected = 1560;
+        Assert.assertEquals(expected, ((int) accountRepository.get(1002)));
+    }
+
+    @Test
+    public void decrease() {
+
+        int amount = (int) accountRepository.get(1002);
+        int expected = amount - 100;
+        accountRepository.decrease(1002, 100.0);
+
+        Assert.assertEquals(expected, (int) accountRepository.get(1002));
+
```

Comment:
Value '1002' is used 3 times, value '100' - 2 times. 
int accountId = 1002;
int decreasedBy = 100;
Because otherwise, it is necessary to understand whether all 1002 mean the same thing or whether it coincided so.